cmd/check-gh-automation/main.go

```diff
@@ -164,10 +181,22 @@ func checkRepos(repos []string, bots []string, ignore sets.Set[string], client a
 			repoLogger.Info("all bots are org members or repo collaborators")
 		}
 
+		if sets.NewString(pluginsForRepo.Plugins...).Has(cherrypickPlugin) {
```

Comment:
I think you removed the wrong part. cherrypick is an external plugin, so we need to check the `ExternalPlugins` for its configuration. Here is an example of where it is configured: https://github.com/openshift/release/blob/1321129519f58b37c31c4add3277b225d12d4902/core-services/prow/02_config/3scale/3scale-operator/_pluginconfig.yaml#L12-L16